field-agent-ui/src/components/Errors.js

```diff
@@ -0,0 +1,17 @@
+function Errors({ errors }) {
+    if (errors.length === 0) {
+        return null;
+    }
+
+    return (
+        <div className="alert alert-danger">
+            <ul>
```

Comment:
When I tried to add an agent, I got the following error message:

TypeError: errors.map is not a function
Errors
src/components/Errors.js:8
   5 | 
   6 | return (
   7 |     <div className="alert alert-danger">
>  8 |         <ul>
     | ^   9 |             {errors.map(error => (
  10 |                 <li key={error}>{error}</li>
  11 |             ))}
View compiled
▶ 18 stack frames were collapsed.
(anonymous function)
src/components/AddAgent.js:73
  70 |         history.push('/agent');
  71 |     } else {
  72 |         // we have error messages
> 73 |         setErrors(data);
     | ^  74 |     }
  75 | })
  76 | .catch(error => console.log(error));
View compiled

Reply:
This is related to not formatting the Date of Birth value as `yyyy-mm-dd`.